<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxSpaceJ">
     <property name="text">
      <string>SpaceJet</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="AirAsia">
     <property name="text">
      <string>AirAsia</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxVistara">
     <property name="text">
      <string>Vistara</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxGoFirst">
     <property name="text">
      <string>GO_FIRST</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxIndigo">
     <property name="text">
      <string>Indigo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_AirInd">
     <property name="text">
      <string>Air_India</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonFilter">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
